### PR TITLE
Added promise definitions for fetchSharedFlow and listSharedflowDeployments

### PR DESCRIPTION
**Pull request by kjshomper**
While the code was in place to fetch a shared flow and list the shared flow deployments, the definitions in the promisesdk were missing.  I've added those and bumped the version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "apigeetool",
-  "version": "0.14.2",
```
Comment (whitlockjc) [inline package.json:3]: Please undo this change, we will bump the `version` as we package/release the next release.

Comment (kjshomper) [reply package.json]: Completed as requested

**Comment by kjshomper** [comment]
Commit f8f32c7454, which was included in the October release to 0.14.x (commit e6a14c2a) created a breaking change when it renamed the SDK function for getkvmmap to getKVMmap in lib/promisesdk.js.  The unit tests were changed to match; however, code dependent on the SDK interface broke.  My next release with the apigeetool-node dependency will mandate at least 0.14.x; so, it will be OK, but wanted you aware as an FYI.  Thanks.

**Comment by whitlockjc** [comment]
The next release will be `0.14.x`, don't worry.  :)  The plan is to merge this, update dependencies and see any other things we can get in.  I'll spend some time on it now.
